packages/@react-spectrum/list/docs/ListView.mdx

```diff
@@ -0,0 +1,351 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/list';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+import packageData from '@react-spectrum/list/package.json';
+
+```jsx import
+import {ActionButton} from '@react-spectrum/button';
+import {ActionGroup} from '@react-spectrum/actiongroup';
+import {ActionMenu} from '@react-spectrum/menu';
+import Delete from '@spectrum-icons/workflow/Delete';
+import Edit from '@spectrum-icons/workflow/Edit';
+import {Flex} from '@react-spectrum/layout';
+import Folder from '@spectrum-icons/workflow/Folder';
+import {Heading, Text} from '@react-spectrum/text';
+import {Image} from '@react-spectrum/image';
+import Info from '@spectrum-icons/workflow/Info';
+import {Item, ListView} from '@react-spectrum/list';
+import {Link} from '@react-spectrum/link';
+```
+
+---
+category: Collections
+keywords: [listview, collections]
+---
+
+# ListView
+
+<PageDescription>{docs.exports.ListView.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Item', 'ListView']} />
+
+## Example
+```tsx example
+<ListView aria-label="List example">
+  <Item textValue="row 1">row 1</Item>
+  <Item textValue="row 2">row 2</Item>
+  <Item textValue="row 3">row 3</Item>
+</ListView>
```

Comment:
To match the other docs, it would be good to have realistic looking data in the ListView examples

Reply:
Good idea, fixed